api/player.py

```
@@ -0,0 +1,14 @@
+from utils.db import get_pitchers_by_team_id, get_hitters_by_team_id
+
+def get_team_players(team_id: int):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)